11ty/definitions/barbaric.md

```diff
@@ -5,7 +5,7 @@ flag:
   text: 'Neo-Colonial/Racist slur'
   level: 'avoid'
 defined: true
-excerpt: something which is obscenely cruel; primitive; unsophisticated.
+excerpt: Something which is obscenely cruel; primitive; unsophisticated. In modern day usage, you can notice this rhetoric being employed almost always for the purpose of demonizing "foreign" (more often than not from the global south) cultures.
```

Comment:
The addition to this excerpt is pulled from the body of the definition. I left the line unmodified for the excerpt, but I am unsure if there are constraints on the length of the excerpt which would require changes.

Reply:
AFAIK, it can vary, but it's around 320 in most instances, I believe.